src/apps/api/views/mixins.py

```diff
@@ -0,0 +1,40 @@
+# NOTE: We don't have delete mixin
+from rest_framework import status
+from rest_framework.response import Response
+
+
+class BulkViewSetMixin(object):
+    def get_serializer_context(self):
+        context = super().get_serializer_context()
```

Comment:
Yes I do believe this is working properly. Without it we would have to send producers on each object from v2. This works because of Python's multiple inheritance I believe, where if one class defines it and the other does not (As long as the order is correct) it falls back to whichever object has it defined.